frontend/src/hooks/api/actions/useChangeRequestApi/useChangeRequestApi.ts

```diff
@@ -13,7 +13,7 @@ export interface IChangeSchema {
         | 'archiveFeature'
         | 'updateSegment'
         | 'addDependency'
-        | 'deleteDependencies';
+        | 'deleteDependency';
```

Comment:
rename to method actually used by backend code. I'd like those types to be taken from orval to prevent this type of errors